mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/api/akka/MasterApiAkkaService.java

```diff
@@ -93,6 +93,7 @@ public class MasterApiAkkaService extends BaseService {
     private final Materializer materializer;
     private final ExecutorService executorService;
     private final CountDownLatch serviceLatch = new CountDownLatch(1);
+    private final boolean namedJobsReferToLaunched;
```

Comment:
nit: Can we wrap this in a config/settings class?

Reply:
umm, you okay if I pass the MasterConfiguration as-is instead of just this property?